src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -430,11 +426,8 @@ p5.RendererGL.prototype._drawImmediateStroke = function() {
   gl.drawArrays(
     gl.TRIANGLES,
     0,
-    this.immediateMode.geometry.lineVertices.length
+    this.immediateMode.geometry.lineVertices.length / 3
```

Comment:
is this guaranteed to always be divisible by three? Do we need to catch any circumstances where it could end up with a float here?

Reply:
I think this should be ok because this is an array filled by internal `p5.Geometry` methods (`_addSegment`, `_addCap`, and `_addJoin`), all of which add x, y, and z values for each point.